COMPATIBILITY.md

```diff
@@ -0,0 +1,17 @@
+# React Native and Expo compatibility
+
+
+| Realm JavaScript       | React Native     | Expo         | npm    | node   |
+|------------------------|------------------|--------------|--------|--------|
+| 11.0.0-rc.2            | 0.70             |              |        |        |
```

Comment:
Is this correct, or does rc2 still support >= 0.66.0? The reason I ask I don't think we added any 0.70-specific APIs, but maybe the ABI changed?

Reply:
Good point. As rc2 isn't released yet, I might have made wrong assumption.

Reply:
I will update once rc2 is released.